Gemfile.lock

```diff
@@ -175,13 +175,13 @@ GEM
     childprocess (4.1.0)
     coderay (1.1.3)
     concurrent-ruby (1.1.10)
-    connection_pool (2.2.5)
+    connection_pool (2.3.0)
```

Comment:
### [connection_pool](https://github.com/mperham/connection_pool)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.5 → 2.3.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/connection_pool/2.2.5/2.3.0)_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/mperham/connection_pool/compare/v2.2.5...v2.3.0).

These are the first 10 commits:
- (e268a8e) [Prefer `require_relative` for internal requires (#152)](https://github.com/mperham/connection_pool/commit/e268a8eb3502341c48c5131615b41e9dc12f8ad4)
- (0870c11) [Use explicit namespaces for `Mutex` and `ConditionVariable` …](https://github.com/mperham/connection_pool/commit/0870c112d65f04e66671287c95c954dc620597a1)
- (d317594) [Drop support for Ruby 2.2 (#154)](https://github.com/mperham/connection_pool/commit/d31759443575b941df81ac89d707eb52ca64511c)
- (15b61ed) [Update gemspec to avoid backticks](https://github.com/mperham/connection_pool/commit/15b61ed859296fea73cd5c019e7ed25ee6d93ba2)
- (c5aef74) [bump](https://github.com/mperham/connection_pool/commit/c5aef742642def23664c4d9c15d12f0786347fb8)
- (7074b31) [Mention new timeout-error class in README (#156)](https://github.com/mperham/connection_pool/commit/7074b317197a04eed76fb786864bb7a79c760cc4)
- (204d9f2) [Use prerelease aware version comparison (#157)](https://github.com/mperham/connection_pool/commit/204d9f24b1fff7f893baf9d26e509bf79ed53083)
- (813c271) [Revert "Use prerelease aware version comparison (#157)"](https://github.com/mperham/connection_pool/commit/813c271e41be0714334834d6517ce68ca1357fc0)
- (02fb930) [Add Ruby 3.1 to CI (#160)](https://github.com/mperham/connection_pool/commit/02fb930fc988b9a9f45011d7413231627a24d497)
- (eb3b2e0) [Add Dependabot for GitHub Actions (#162)](https://github.com/mperham/connection_pool/commit/eb3b2e0d2e41b71def1ae86533f038519a9c1060)


</details>